### PR TITLE
Added Domain Objects for DoseEvent, DoseMedication, and DoseSeries

### PR DESCRIPTION
**Pull request by geoffreygraddy**
Created domain objects for DoseEvent, DoseMedication, and DoseSeries.  These objects will be used in the Dose page which will be in later commits.

### CODE REVIEW

File: app/dose-medication/dose-medication.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    user: DoseAmigosUser;
+    rxcui: number;
+    name: string;
+    doseAmount: number;
+    doseUnit: string;
+    totalAmount: number;
+    doseInstructions: string;
+    firstTaken: Date;
+    lastDoseTaken: Date;
+    nextScheduledDost: Date;
```
Comment (mckoon) [inline app/dose-medication/dose-medication.ts]: Typo `Dost` -> `Dose`

- [ ] Fix nextScheduledDose typo

**Comment by mckoon** [comment]
Your commit message should reference Issue #23 and could even say `Closes #23` in it if you want merging this PR to automatically close the issue for you.

**Comment by mckoon** [comment]
Looks good to me, just fix the typo.

**Comment by mckoon** [comment]
+1

**Comment by jleopold28** [comment]
👍

**Comment by jking31cs** [comment]
👍

**Pull request merged**